Makefile.am

```diff
@@ -1,6 +1,10 @@
 ACLOCAL_AMFLAGS = -I m4
 
-SUBDIRS = include libjwt examples tests dist
+SUBDIRS = include libjwt
+if BUILD_EXAMPLES
+SUBDIRS += examples
+endif
+SUBDIRS += tests dist
```

Comment:
You can actually leave "tests dist" in the original SUBDIRS declaration. There's no reason for the ordering, examples can come last.

Reply:
@benmcollins I added that change -- was just following the doctrine _primum non nocere_ -- first do no harm.